### PR TITLE
Image and input

### PR DESCRIPTION
**Pull request by ChhikaraBRUH**
Added Image and Input Component. Please approve the pull request.

### CODE REVIEW

**Review by mohammed-sas — APPROVED** [review]
your code is fine

**Review by gadgetfather — APPROVED** [review]
code is neatly written!

**Review by Husain01 — APPROVED** [review]
All the components working completely fine.

File: Components/Image/image.html [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+				<br />
+				<iframe
+					src="https://carbon.now.sh/embed?bg=rgba%28255%2C255%2C255%2C1%29&t=seti&wt=sharp&l=htmlmixed&width=522.25&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=false&pv=0px&ph=0px&ln=false&fl=1&fm=Hack&fs=15.5px&lh=133%25&si=false&es=2x&wm=false&code=%253Cimg%2520src%253D%2522...%2522%2520class%253D%2522img-circle%2522%2520alt%253D%2522...%2522%253E"
+					style="
+						width: 522px;
+						height: 95px;
+						border: 0;
+						transform: scale(1);
+						overflow: hidden;
+					"
```
Comment (vp5h) [inline Components/Image/image.html:123]: This is kind of repetitive, we can use 2-3 classes for better readability

**Review by vp5h — APPROVED** [review]
Code is great, Just deploying it would save time while reviewing, Keep up the good work!

**Comment by AnkurChunekar** [comment]
Great work, Code looks clean and works fine.

**Review by AnkurChunekar — APPROVED** [review]

File: Components/Image/image.html (hunk 1 above)
Comment (ChhikaraBRUH) [reply Components/Image/image.html:123]: This is the default iframe code that I copied from carbon.now.sh
Yes I can add styling classes, but then I would I have to manually add those classes every time. This is just for the documentation purposes.
Thank you for kind inputs.

**Comment by ChhikaraBRUH** [comment]
> Code is great, Just deploying it would save time while reviewing, Keep up the good work!

I will deploy the index page asap.

**Pull request merged**